### ISSUE TITLE
Switch release hosting from SourceForge to GitHub

### ISSUE THREAD

**Issue by remram44** [VisTrails/VisTrails#1129]
SourceForge suffers from a very bad reputation lately, because of some prominent OSS projects being rebundled with adware ([GIMP](http://arstechnica.com/information-technology/2015/05/sourceforge-grabs-gimp-for-windows-account-wraps-installer-in-bundle-pushing-adware/), [VLC](https://blog.l0cal.com/2015/06/02/what-happened-to-sourceforge/), [Nmap](http://arstechnica.com/information-technology/2015/06/black-mirror-sourceforge-has-now-siezed-nmap-audit-tool-project/)...).

Trying it today, downloading from GitHub is also a lot faster.

One issue I see is the download statistics, GitHub only offers a single download counter per file, accessible through the API (but pulling that regularly to make graphs is easy).

**Comment by dakoop** [comment]
Wow, I've seen migration away from SF, but I hadn't read those stories. Sad.

**Comment by rexissimus** [comment]
I agree. We would then not need to keep a separate file-hoster. We may want to keep older releases there? But the adware seems to be added to stale accounts so this may not be a good idea.

Hosting the files on GitHub seems reasonable, but does this look like it would work in the long term? I think other repo-hosting sites (and also github?) have disabled binary uploads before? But if that happens it should not be hard to migrate somewhere else.

**Comment by remram44** [comment]
GitHub [disabled](https://github.com/blog/1302-goodbye-uploads) the "downloads" feature before [rolling out](https://github.com/blog/1547-release-your-software) the "releases" feature. But there was a 6 month gap, it's true.
